### PR TITLE
Remove stale content from ways-to-contribute.md

### PR DESCRIPTION
**Pull request by mishmosh**
# Describe your changes
- Remove links to archived repos
- Remove references to Protocol Labs-specific things
- Added Ecosystem directory link

# Files changed
ways-to-contribute.md

# What issue(s) does this address?
n/a

# Does this update depend on any other PRs?
no

## Checklist before requesting a review
- [ ] Passing the beta version of the **Check Markdown links for modified files** check. Action results can be viewed [here](https://github.com/ipfs/ipfs-docs/actions/workflows/action.yml).

## Checklist before merging
- [ ] Passing all required checks (The beta **Check Markdown links for modified files** check is not required)

### CODE REVIEW

**Comment by mishmosh** [comment]
@johnnymatthews can you help me decipher the 2 failing vale tests?

File: docs/community/contribute/ways-to-contribute.md [hunk 1]
```diff
@@ -45,45 +45,33 @@ If you have never contributed to an open-source project before or just need a re
...
 - [ipfs/awesome-ipfs](https://github.com/ipfs/awesome-ipfs)
-- [ipfs/notes](https://github.com/ipfs/notes)
-- [ipfs/apps](https://github.com/ipfs/apps)
-- [IPFS forum](https://discuss.ipfs.tech/)
+- [IPFS Forum](https://discuss.ipfs.tech/)
 
 ## Protocol Design
 
-IPFS is ultimately about building better protocols, and we always welcome ideas and feedback on how to improve those protocols. Post feedback, issues, and proposals in the `specs` projects in our Github organizations or in [`ipfs/notes`](https://github.com/notes).
+IPFS is ultimately about building better protocols, and we always welcome ideas and feedback on how to improve those protocols. Post feedback, issues, and proposals via Github.
```
Comment (ElPaisano) [inline docs/community/contribute/ways-to-contribute.md:71]:
```suggestion
IPFS is ultimately about building better protocols, and we always welcome ideas and feedback on how to improve those protocols. Post feedback, issues, and proposals via GitHub.
```

Comment (ElPaisano) [inline docs/community/contribute/ways-to-contribute.md:71]:
```suggestion
IPFS is ultimately about building better protocols, and we always welcome ideas and feedback on how to improve those protocols. Post feedback, issues, and proposals via GitHub.
```

**Comment by ElPaisano** [comment]
@mishmosh this PR is good to go, the failing tests were just the spell checker. It  flagged "Github", which I updated to "GitHub". Merging

**Pull request merged**